src/components/inputs/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+import {Icon} from '../../generic';
+
+/**
+ * Displays a checkbox
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {boolean} [props.disabled]  Flag indicating whether checkbox is disabled
+ * @param {string} props.name  Checkbox name
+ * @param {Function} [props.onChange]  Action to perform on toggle change
+ * @param {boolean} [props.required]  Flag indicating whether checkbox is required
+ * @param {string} props.value  Checkbox value
+ * @param {boolean} props.selected Flag indicating whether checkbox is selected
+ * @returns {object} JSX of the element
+ */
+export default function Checkbox({
+  className,
+  disabled,
+  name,
+  onChange,
+  required,
+  selected,
+  value,
+}) {
+  const enabled = !disabled;
+  const SPACE_KEY = ' ';
+  const ENTER_KEY = 'Enter';
+
+  const [cssClasses, sc] = webexComponentClasses('checkbox', className, {
+    check: selected,
+    uncheck: !selected,
+    disabled,
+    enabled,
+  });
+
+  return (
+    <div
+      role="checkbox"
+      aria-label={`${name}.${value}`}
```

Comment:
aria label should be taken from a dedicated `ariaLabel` prop (optional) just like for InputField